src/main/kotlin/no/nav/tilleggsstonader/sak/vilkår/vilkårperiode/evaluering/EvalueringMålgruppe.kt

```diff
@@ -61,14 +118,12 @@ object EvalueringMålgruppe {
         null -> ResultatDelvilkårperiode.IKKE_VURDERT
     }
 
-    val IMPLISITT_OPPFYLT_MÅLGRUPPE = ResultatEvaluering(
-        DelvilkårMålgruppe(
-            medlemskap = Vurdering(
-                svar = SvarJaNei.JA_IMPLISITT,
-                begrunnelse = null,
-                resultat = ResultatDelvilkårperiode.OPPFYLT,
-            ),
-        ),
-        resultat = ResultatVilkårperiode.OPPFYLT,
-    )
+    val IMPLISITT_OPPFYLT_MÅLGRUPPE =
+        Vurdering(
+            svar = SvarJaNei.JA_IMPLISITT,
+            begrunnelse = null,
+            resultat = ResultatDelvilkårperiode.OPPFYLT,
+        )
+
+    private fun ugyldigSvarDekketAvAnnetRegelverkFeilmelding(svar: SvarJaNei?, type: MålgruppeType) = "Ugyldig svar=$svar for dekket av annet regelverk for $type"
```

Comment:
Dette er en lang rad
```suggestion
    private fun ugyldigSvarDekketAvAnnetRegelverkFeilmelding(svar: SvarJaNei?, type: MålgruppeType) =
        "Ugyldig svar=$svar for dekket av annet regelverk for $type"
```